api/db/mongo/mongo.go

```diff
@@ -41,18 +41,19 @@ type Database interface {
 }
 
 // Connect connects to mongo and returns the session.
-func Connect() error {
+func Connect(address, dbName, username, password string, poolLimit, port int, timeout time.Duration) error {
 
-	log.Info("Connect", "DB", 21)
-	mongoConfig := config.APIConfiguration.MongoDBConfig
+	values := fmt.Sprintf("Mongo received vars: Address - %s\nDBName - %s\nUsername - %s\nPassword - %s\nPoolLimit - %d\nPort - %d \nTimeout - %v\n", address, dbName, username, password, poolLimit, port, timeout)
```

Comment:
Ops, it looks like sensitive data is being sent to the logging system! 😅 Would you mind removing it, please?

Reply:
Ok! Changes have been made in this [commit](https://github.com/globocom/huskyCI/pull/400/commits/a4a6afe14bd719d3281c3d1de12be4a32b31e575).